test/integration/ferc1_eia_train_test.py

```diff
@@ -140,22 +153,23 @@ def test_validate_override_fixes(
         start_date=min(eia_ferc1.report_date),
         end_date=max(eia_ferc1.report_date),
     )
-    validate_override_fixes(
-        validated_connections=test_df,
-        ppe=plant_parts_eia,
-        eia_ferc1=eia_ferc1,
-        training_data=eia_ferc1_training_data_restricted,
-        expect_override_overrides=False,
-        allow_mismatched_utilities=False,
-    )
+    with expectation:
+        validate_override_fixes(
+            validated_connections=test_df,
+            ppe=plant_parts_eia,
+            eia_ferc1=eia_ferc1,
+            training_data=eia_ferc1_training_data_restricted,
+            expect_override_overrides=False,
+            allow_mismatched_utilities=False,
+        )
```

Comment:
Use pytest.raises() and the context manager to communicate more specific test outcomes, rather than marking xfail -- these aren't failures, they're tests that verify an exception is raised appropriately.